src/vip/batch_address_test_tool/processor.clj

```diff
@@ -63,7 +63,10 @@
 
 (defn retrieve-polling-locations*
   [ctx]
-  ctx)
+  (let [addresses (:addresses ctx)
+        polling-locations (doall (map #(civic-info/address->polling-location (:address %)) addresses))
```

Comment:
So the `ctx` is a map that has `:addresses` as a vector of all of the input addresses.  The `map` function here takes the addresses and calls `address->polling location` on each of them generating a new vector of the output of the call to the Civic Info API which is merged together with the original input address in the next line.  What can we change to make this more clear?